### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by adamledwards**
# Home Page
This PR adds static content for the home page. The markup is based on the GOV.UK design system

image
image

### CODE REVIEW

File: src/site/_includes/components/navigation/macro.njk [hunk 1]
```diff
@@ -1,9 +1,9 @@
-{% macro guidanceNavigation(params) %}
+{%- macro guidanceNavigation(params) -%}
```
Comment (adamledwards) [inline src/site/_includes/components/navigation/macro.njk:1]: Added whitespace control that was missed from my last pr

File: src/site/index.scss [hunk 2]
```diff
@@ -1,5 +1,11 @@
 @use '@moduk/frontend/src/css' with (
-  $govuk-new-link-styles: true
+  $govuk-new-link-styles: true,
+  // light maroon
+  $govuk-link-colour: #643f58,
+  // dark maroon
+  $govuk-link-hover-colour: #4b263e,
+  // maroon
+  $govuk-link-active-colour: #532a45
```
Comment (adamledwards) [inline src/site/index.scss:8]: We have and chicken and egg scenario here, I wanted to override the `gov-links` colours but I don't have access to`gov-colour('maroon')` at this point. Any suggestion?

Comment (jpveooys) [reply src/site/index.scss:8]: Don't think we can do it that way as that will affect the components. I'm using a `moduk-link--light-maroon` modifier class in #29.

**Review by jpveooys — COMMENTED** [review]
Nice!

Couple of comments:

- we should probably also add the footer since it's part of the page
- we don't have a styles page either, maybe putting the design library text in without the link would be better for now

File: src/site/_includes/components/updates/index.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@use '@moduk/frontend/src/css' as moduk;
+
+.guidance-updates {
+  background-color: moduk.govuk-colour('light-grey');
+  border-bottom: 1px solid moduk.govuk-colour('mid-grey');
```
Comment (jpveooys) [inline src/site/_includes/components/updates/index.scss:5]: Might be good to use ` $govuk-border-colour` for this one:

```suggestion
  border-bottom: 1px solid moduk.$govuk-border-colour ;
```

File: src/site/index.njk [hunk 4]
```diff
@@ -19,7 +20,38 @@ order: 0
...
+        ]
+      ) -}}
+
+      <div class="govuk-grid-column-full">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-8">
+      </div>
+
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h2 id="support" class="govuk-heading-m">Support</h2>
+        <p class="govuk-body">Email the MOD.UK Frontend team at <a class="govuk-link moduk-link--maroon" href="mailto://design-system@digital.mod.uk">design-system@digital.mod.uk</a> if you have questions or feedback.
```
Comment (jpveooys) [inline src/site/index.njk:51]: Small mistake in the `mailto` link:

```suggestion
        <p class="govuk-body">Email the MOD.UK Frontend team at <a class="govuk-link moduk-link--maroon" href="mailto:design-system@digital.mod.uk">design-system@digital.mod.uk</a> if you have questions or feedback.
```

**Review by jpveooys — APPROVED** [review]

**Pull request merged**